src/Simplify.cpp

```diff
@@ -436,8 +436,27 @@ class Simplify : public IRMutator2 {
         return is_round_up_div(mul->a, *factor);
     }
 
+    struct VisitResult {
+        Expr expr;
+        bool done;
+        template<typename T>
+        VisitResult(T &&t) : expr(std::forward<T>(t)), done(true) {}
+    };
+
+    template<typename T>
+    VisitResult remutate(T &&t) const {
```

Comment:
I want to pronounced this "rem-you-tate"

Reply:
"revisit" was another thing I considered.